front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -83,7 +83,7 @@ export class CreateF3XStep1Component extends DestroyerComponent implements OnIni
       .pipe(takeUntil(this.destroy$))
       .subscribe(([committeeAccount, existingCoverage]) => {
         this.committeeAccount = committeeAccount;
-        const filingFrequency = this.userCanSetFilingFrequency ? 'Q' : committeeAccount?.filing_frequency;
+        const filingFrequency = committeeAccount?.filing_frequency ?? 'Q';
```

Comment:
looks like the acs are a little more complicated:
`if Filing Frequency = Q, A or T, system will default to 'Quarterly Reporting'`

Reply:
Ah I see what you mean, we wouldn't want to set filingFrequency to 'T'. I updated this logic so now everything that isn't 'M' defaults to 'Q'. That way 'Q', 'A', and 'T' all go to 'Q', but also when it's undefined it goes to 'Q' as well.